Library/Homebrew/requirements/arch_requirement.rb

```diff
@@ -14,21 +14,37 @@ class ArchRequirement < Requirement
   attr_reader :arch
 
   def initialize(tags)
-    @arch = tags.shift
+    @arch = tags
     super(tags)
```

Comment:
This isn't quite right. You want to remove the arch tags but still leave e.g. `:build` in place so you'll still need `.shift` or equivalent.

Reply:
Oh, I see - I must have misinterpreted what "tags" are in this context! 😅 

Thanks for spotting it and explaining.